### ISSUE TITLE
Investigate if integrations info parameter is being sent correctly

### ISSUE THREAD

**Issue by TheUnderScorer** [fingerprintjs/fingerprintjs-pro-svelte#19]

**Comment by makma** [comment]
When sending the identification request, the package name in the `ii` query string parameter contains a URL encoded (`%40`) at the sign (`@`).

Current value: `https://api.fpjs.io/?ci=js/3.8.1&ii=%40fingerprintjs/fingerprintjs-pro-svelte/1.0.0&ii=fingerprintjs-pro-spa/0.4.1`
Expeced value: `https://api.fpjs.io/?ci=js/3.8.1&ii=fingerprintjs/fingerprintjs-pro-svelte/1.0.0&ii=fingerprintjs-pro-spa/0.4.1`

**Comment by necipallef** [comment]
[Here](https://github.com/fingerprintjs/fingerprintjs-pro-svelte/blob/main/src/lib/options.dist.ts#L11) is the source of the bug. I think we shouldn't use the package name as the `ii` parameter, we should create another constant value specifically for this purpose, like below:
`const integrationInfoKey = 'fingerprintjs/fingerprintjs-pro-svelte'`

**Comment by TheUnderScorer** [comment]
@necipallef it doesn't seem to be the source. Here's how the generated file looks like:
```
import type { FpjsClientOptions } from '@fingerprintjs/fingerprintjs-pro-spa';

const pkgVersion = '1.1.2';
const pkgName = 'fingerprintjs-pro-svelte';

export function getOptions(options: FpjsClientOptions) {
  const clientOptions: FpjsClientOptions = {
    ...options,
    loadOptions: {
      ...options.loadOptions,
      integrationInfo: [...(options.loadOptions?.integrationInfo ?? []), `${pkgName}/${pkgVersion}`],
    },
  };

  return clientOptions;
}
```

The tokens are replaced like this:
```
const tokens = Object.entries({
  __PACKAGE_VERSION__: pkg.version,
  __PACKAGE_NAME__: pkg.name.split('/')[1],
});
```

so the `@` shouldn't even exist in here :smile:

**Comment by TheUnderScorer** [comment]
@makma the issue that you mentioned existed previously (introduced in this commit: https://github.com/fingerprintjs/fingerprintjs-pro-svelte/commit/5152bcf2e09791a39a8250e7e6213d4e115e5ea7#diff-b52c72805d25139dbb98c8683c818134b6a29cfdbd1e2be4a7df700a6fb0f68d) but was fixed in [1.1.1,](https://github.com/fingerprintjs/fingerprintjs-pro-svelte/releases/tag/v1.1.1).

I suppose the reason for missing data in retool could be people that still use the old version of this library before the fix.
